src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -383,6 +411,8 @@ function ReportActionItemMessageEdit(props) {
                             onEmojiSelected={addEmojiToTextBox}
                             nativeID={emojiButtonID}
                             emojiPickerID={props.action.reportActionID}
+                            // Keep focus on the composer when emoji button is clicked.
+                            onMouseDown={(e) => e.preventDefault()}
```

Comment:
From last PR I don't think we want this for the emoji button because it opens a modal and messes up with focus trap. Can you double check?

Reply:
Yes, I restored